odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -949,7 +987,53 @@ PROCEDURE READCONFIGFILESETTINGS
  FOR TMPLOOP=0 TO MAXSLTYPES
   DO
    VALIDSHARETYPES(TMPLOOP)=LELIST(TMPLOOP)
-  END
+
+[* If share type is eligible, then loop through the share types
+** defined in the OTA parameter setting and if a matching share type
+** has been defined, leave the OTA amount at that value, else if the
+** OTA default amount has been defined, set the OTA amount to that
+** value, else, leave it at the initialization value of $0.00.
+*]
+   IF VALIDSHARETYPES(TMPLOOP)=TRUE THEN
+    DO
+     SHARETYPEFOUND=FALSE
+     OTAAMOUNT=$0.00
+     FOR OTALOOP=1 TO PARAMOTACOUNT
+      DO
+       IF PARAMOTASHARETYPE(OTALOOP)=TMPLOOP THEN
+        DO
+         SHARETYPEFOUND=TRUE
+         OTAAMOUNT=PARAMOTAAMOUNT(OTALOOP)
+        END
+      END [FOR OTALOOP=1 TO PARAMOTACOUNT]
+
+     BNODEBUGMSG=FORMAT("Evaluating OTA param list for share type 9999",TMPLOOP)+
+                 "  SHARE TYPE FOUND="+BNOTRUEFALSECHR(SHARETYPEFOUND)
+     CALL BNODEBUGMSGADD
+
+     IF SHARETYPEFOUND=FALSE AND
+        DEFAULTOTAVALUE>=$0.00 THEN
+      OTAAMOUNT=DEFAULTOTAVALUE
+
+     PARAMOTAAMOUNT(OTALOOP)=OTAAMOUNT
+
+     TMPCHR=FORMAT("###,###,##9.99",OTAAMOUNT)
+     CALL NLS
+     OTAAMOUNTCHR=TMPCHR
+
+     BNODEBUGMSG=FORMAT("VALID SHARE TYPE: 9999 - OTA PARAM FOUND: ",TMPLOOP)+
+                 BNOTRUEFALSECHR(SHARETYPEFOUND)
+     IF SHARETYPEFOUND=TRUE THEN
+      BNODEBUGMSG=BNODEBUGMSG+" FOR: $"+OTAAMOUNTCHR+".  "+
```

Comment:
@tkainz For line 1027 OTAAMOUNTCHR is printed, then the same variable, OTAAMOUNTCHR,  is printed again in line 1028, both as part of the same debug line. Did you mean to use a different variable for one of them?